core/src/test/java/org/elasticsearch/common/joda/DateMathParserTests.java

```diff
@@ -304,8 +304,21 @@ public void testThatUnixTimestampMayNotHaveTimeZone() {
             parser.parse("1234567890123", () -> 42, false, DateTimeZone.forTimeZone(TimeZone.getTimeZone("CET")));
             fail("Expected ElasticsearchParseException");
         } catch(ElasticsearchParseException e) {
-            assertThat(e.getMessage(), containsString("failed to parse date field"));
-            assertThat(e.getMessage(), containsString("with format [epoch_millis]"));
+            assertThat(e.getMessage(), containsString("Format [epoch_millis] only supports UTC time zones"));
         }
+
+        // Explicitly parsing a UTC timestamp is acceptable
+        parser.parse("1234567890123", () -> 42, false, DateTimeZone.forTimeZone(TimeZone.getTimeZone("UTC")));
+
+        parser = new DateMathParser(Joda.forPattern("epoch_second"));
+        try {
```

Comment:
The modern idiom we use for this now is `expectThrows`.

If I were doing this I'd convert the one above to `expectThrows` as well.